src/components/config/KeyboardShortcutsPanel.tsx

```diff
@@ -0,0 +1,117 @@
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import Paper from "@mui/material/Paper";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer/TableContainer";
+import TableRow from "@mui/material/TableRow/TableRow";
+import { type } from "@tauri-apps/api/os";
+import { Component } from "react";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+type Props = object;
+
+type State = object;
+
+let isMac = false;
+// We don't want to block during the imports stage, especially
+// if type() somehow fails.
+type().then(type=>{
+    isMac = (type == "Darwin");
+})
+// These default to the Mac ⌘ key because it's easier to replace with Ctrl than vice versa
+let shortcuts = {
+  Path: {
+    "G, ⌘ + G": "Generate current path",
+    Space: "Start/stop path playback",
+    Escape: "Clear sidebar and navbar selection"
+  },
+  "Field View": {
+    "Left Mouse Drag": "Pan around the field",
+    "Scroll Wheel, Trackpad Scroll": "Zoom in or out of the field",
+    "⌘ + Plus": "Zoom in",
+    "⌘ + Minus": "Zoom out",
+    "⌘ + 0": "Fit zoom to waypoints"
+  },
+  File: {
+    "⌘ + N": "New File",
+    "⌘ + O": "Open File",
+    "⌘ + Z": "Undo",
+    "⌘ + Shift + Z, ⌘ + Y": "Redo"
+  },
+  Waypoints: {
+    "1 through 4": "Select waypoint type in navbar",
+    "Shift-1 through Shift-4": "Change type of selected waypoint",
+    "Left Mouse Click":
+      "Add waypoint type selected in navbar at click position",
+    "Left Arrow, X": "Select previous waypoint in path",
+    "Right Arrow, Z": "Select next waypoint in path",
+    "Delete/Backspace": "Delete selected waypoint"
+  },
+  "Selected Waypoint Movement": {
+    "Left Mouse Drag on Center": "Move Waypoint",
+    "Left Mouse Drag on Edge": "Rotate Waypoint",
+    E: "Rotates 1/32 turn clockwise",
+    Q: "Rotates 1/32 turn counterclockwise",
+    "Shift + Rotation Key": "Rotate by 1/8 turn instead of 1/32 turn",
+    W: "Translate 0.1 m up (+Y)",
+    A: "Translate 0.1 m left (-X)",
+    S: "Translate 0.1 m down (-Y)",
+    D: "Translate 0.1 m right (+X)",
+    "Shift + Translation Key": "Translate by 0.5 m instead of 0.1 m"
+  }
+};
+
+class KeyboardShortcutsPanel extends Component<Props, State> {
+  rowGap = 16;
+  render() {
+    return (
+      <div
+        style={{
+          fontSize: "2rem",
+          marginTop: `${1 * this.rowGap}px`,
+          marginBottom: `${1 * this.rowGap}px`,
+          marginInline: "auto",
+          width:"max-content"
+        }}
+      >
+        {Object.entries(shortcuts).map((entry) => (
+          <>
+            <Accordion>
```

Comment:
What is this gap doing here?

<img width="953" alt="98105" src="https://github.com/SleipnirGroup/Choreo/assets/70717139/fc5e2587-0577-420a-a0e1-68e4ef599d3c">

Reply:
Fixed